Babylon/commands/__init__.py

```diff
@@ -3,15 +3,15 @@
 from .api import api
 from .azure import azure
 from .config import config
-from .terraform_cloud import terraform_cloud
-from .working_dir import working_dir
+from .vault import vault
+from .git_hub import github
 
 list_groups = [
-    webapp,
-    powerbi,
-    terraform_cloud,
```

Comment:
There is no more 'terraform-cloud' commands ?

Reply:
oups!